packages/conf-readline/conf-readline.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: ["vincent.tourneur@inria.fr"]
+authors: ["sylvain.pogodalla@inria.fr" "vincent.tourneur@inria.fr"]
+depends: [
+  "conf-pkg-config" {build}]
+depexts: [
+  ["libreadline8" "libreadline-dev"] {os-family = "debian"}
+  ["libreadline8" "libreadline-dev"] {os-family = "ubuntu"}
+  ["readline" "readline-devel"] {os-distribution = "centos"}
+  ["readline" "readline-devel"] {os-distribution = "fedora"}
+  ["readline" "readline-dev"] {os-family = "alpine"}
```

Comment:
I believe the dev packages already require the library itself (that would be a bug otherwise)
```suggestion
  ["libreadline-dev"] {os-family = "debian"}
  ["libreadline-dev"] {os-family = "ubuntu"}
  ["readline-devel"] {os-distribution = "centos"}
  ["readline-devel"] {os-distribution = "fedora"}
  ["readline-dev"] {os-family = "alpine"}
```